### PR TITLE
overhaul sre leveling guide and fix glitches in the sre challenge descriptions

### PR DESCRIPTION
**Pull request by knisbet**
I tried to take some influence from the excellent guide dropbox has around leveling (they have 7 levels instead of our 6), and go pretty deep into lots of different areas, so I tried to simplify the guidance a bit and stay similar to how @klizhentas defined developer skills/roles. https://dropbox.github.io/dbx-career-framework/overview.html

Also, did a bit of cleanup that's been missing for awhile.

### CODE REVIEW

File: challenges/platform/README.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+* Progressing beyond minimum viable monitoring and alerting and fully adopting Grafana Cloud as our centralized metrics, alerting and logging platform.
+
+The toolset we currently use is:
+* Golang
+* Terraform and Packer
+* Kubernetes
+* Prometheus / Alertmanager / Loki internally and on Grafana Cloud
+* Amazon Web Services
+* Drone.io
+* and More
```
Comment (michaelmcallister) [inline challenges/platform/README.md:21]: nit: 'More' -> 'more'

**Review by michaelmcallister — APPROVED** [review]
LGTM

**Review by rcanderson23 — APPROVED** [review]
Addition of the toolset and current goals is a good addition. LGTM

**Review by logand22 — APPROVED** [review]
LGTM

**Pull request merged**